### PR TITLE
add carousel with european cities with breakpoint

### PR DESCRIPTION
**Pull request by cartige**

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+  margin-bottom: 10%;
+ }
+
 .map {
   background-color: rgba(228, 228, 228, 0.486);
   padding: 1rem;
   border-radius: 3%;
   width: fit-content;
-}
+ }
```
Comment (LoicBrassart) [inline frontend/src/App.css]: Hmm ? 🧐 
How did you add a single blank space here ? Prettier should have caught/fixed that

File: frontend/src/components/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable import/no-unresolved */
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx]: Wowowow, why do you want to disable an eslint rule ?
This rule specifies your imports should be functional, I don't see any case where it'd be a good idea to disable it

File: frontend/src/components/carousel.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  display: -webkit-flex;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-justify-content: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  -webkit-align-items: center;
+  align-items: center;
```
Comment (LoicBrassart) [inline frontend/src/components/carousel.css]: I *really* doubt you need all these properties

File: frontend/src/components/carousel.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  bottom: 20px;
+  right: 20px;
+  background-color: black;
+  color: white;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.swiper-pagination-bullet-active {
+  background-color: #000 !important;
```
Comment (LoicBrassart) [inline frontend/src/components/carousel.css]: I don't like the `!important`. Why did you have to use it ?

**Review by LoicBrassart — COMMENTED** [review]
A few questions, please !

**Review by CharlyJuteau — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
There remains a conflict, plz fix it before assigning me ;-)

**Review by CharlyJuteau — APPROVED** [review]

File: frontend/src/components/Carousel.jsx [hunk 5]
```diff
@@ -1,8 +1,10 @@
-/* eslint-disable import/no-unresolved */
 import { Swiper, SwiperSlide } from "swiper/react";
 
+// eslint-disable-next-line import/no-unresolved
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel.jsx:3]: Please fix like this:
```
// eslint-disable import/no-unresolved
import "swiper/css";
import "swiper/css/pagination";
import "swiper/css/navigation";
// eslint-enable import/no-unresolved
```

File: frontend/src/components/Mapping/Mapping.jsx [hunk 6]
```diff
@@ -1,6 +1,8 @@
+// eslint-disable-next-line import/no-unresolved
```
Comment (LoicBrassart) [inline frontend/src/components/Mapping/Mapping.jsx:1]: The problem shouldn't appear with Leaflet: doesn't it work as intended ?

**Review by LoicBrassart — APPROVED** [review]
Good job! 
You can merge it as is, but you could also fix a few other things (nicer syntax)

**Pull request merged**